src/test/java/org/opensearch/neuralsearch/processor/ScoreNormalizationIT.java

```diff
@@ -156,7 +160,7 @@ public void testL2Norm_whenOneShardAndQueryMatches_thenSuccessful() {
      *         {
      *             "normalization-processor": {
      *                 "normalization": {
-     *                     "technique": "l2"
+     *                     "technique": "min_max"
```

Comment:
why this change? looks like test is for l2

Reply:
It's for [min max](https://github.com/opensearch-project/neural-search/pull/1224/files#diff-03f340a64b84233c32a824c2ce968e980a14d1188724571f4b9108dadbc6ad8cR174)